src/openapi/openapi.yml

```diff
@@ -2,12 +2,399 @@ openapi: '3.0.2'
 info:
   title: Resource Hub
   version: '0.1'
+
 servers:
   - url: /v0
+
+components:
+  schemas:
+    ID:
+      type: integer
+      minimum: 1
+    Name:
+      type: string
+      pattern: '^\w+$'
+
+    Error:
+      type: object
+      required:
+        - error
+      properties:
+        error:
+          type: string
+
+    Quota:
```

Comment:
I like having quota here but the quota we have isn't global, it is region-based. Each region has its own quota for each user. Not sure how we should tackle this schematic @brhatiga ?

Reply:
This is schema definition, it doesn't work like global quota, it is something like "class Quota" that can be used on region/cluster/user/... wherever we will need it. Currently it is used on region ([line 86](https://github.com/fpob/rhub-api/blob/api-draft/src/openapi/openapi.yml#L86)).